pkgs/by-name/at/atar/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "atar";
+  version = "0.1.24";
```

```suggestion
  version = "0.1.25";
```
You just released a new tag